.travis.yml

```diff
@@ -37,7 +50,24 @@ matrix:
        - cmake --install . --config Release
        - popd
      script:
-       - python setup.py build
+       - python setup.py bdist
+       - PyInstaller --onefile --distpath dist/win32 --paths=build/lib.win32-$pyverbrief --add-data=src/rdiff_backup.egg-info/PKG-INFO\;rdiff_backup.egg-info --console build/scripts-$pyverbrief/rdiff-backup
+     before_deploy:
+       - cp CHANGELOG COPYING README.md docs/FAQ.md docs/examples.md docs/DEVELOP.md docs/Windows-README.md dist/win32
+       - pushd dist/win32
+       - version=`./rdiff-backup.exe --version | awk '{ print $2 }'`
```

Comment:
Using `python setup.py --version` you would not need to "parse" the version using awk.